<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_reservation_form</class>
 <widget class="QDialog" name="customer_reservation_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID No.: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkin Date: </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="checkin_date">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2082</year>
     <month>8</month>
     <day>13</day>
    </date>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Room No.: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Type: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contact">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="alipay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Alipay</string>
   </property>
  </widget>
  <widget class="QComboBox" name="id_type">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mainland ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Passport</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone No.: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_number">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>380</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkout Date: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="gender">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="credit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit Card</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="room_number">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="checkout_date">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2082</year>
     <month>8</month>
     <day>13</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="bank_card">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Debit Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="wechat">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wechat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Method: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reservation Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>520</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>customer_reservation_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
